components/atoms/details2/__tests__/stories.js

```diff
@@ -0,0 +1,175 @@
+    </ButtonToolbar>
+  )
+
+  const startActionComponent = <Button>button</Button>
+
+  const endActionComponent = <Button>lastButton</Button>
+
+  return (
+    <>
+      <Details2
+        content={
+          <>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <Button
+              onClick={() => {
+                DetailsCommunicationService.send({
+                  name: MessageNames.DetailsComponent.SET_OPEN,
+                  id: 'danger_Details',
+                  payload: true
+                })
+              }}
+            >
+              Open red Details
+            </Button>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+          </>
+        }
+        context='primary'
+        endActionComponent={endActionComponent}
+        open={false}
+        startActionComponent={startActionComponent}
+        style={{ marginBottom: '2rem' }}
+        title='first details component'
+        toolbar={toolbar}
+        uniqueId='primary_Details'
+        unmountContentOnClose
+      />
+      <Details2
+        content={
+          <>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+            <Button
+              onClick={() => {
+                DetailsCommunicationService.send({
+                  name: MessageNames.DetailsComponent.SET_OPEN,
+                  id: 'primary_Details',
+                  payload: true
+                })
+              }}
+            >
+              Open primary Details
+            </Button>
+            <p>wdsfsadfgsafgd</p>
+            <p>wdsfsadfgsafgd</p>
+          </>
+        }
+        context='danger'
+        endActionComponent={endActionComponent}
+        open
+        startActionComponent={startActionComponent}
+        style={{ marginBottom: '2rem' }}
+        title='second details component'
+        toolbar={toolbar}
+        uniqueId='danger_Details'
+      />
+      <Row>
+        <Column md={6}>
+          <Details2
+            context='warning'
+            open={false}
+            startActionComponent={
+              <Button
+                size='sm'
+                onClick={() => {
+                  DetailsCommunicationService.send({
+                    name: MessageNames.DetailsComponent.SET_OPEN,
+                    id: 'success_Details',
+                    payload: false
+                  })
+                }}
+              >
+                close success Details
+              </Button>
+            }
+            title='third details component'
+            uniqueId='warning_Details'
+          />
```

Comment:
I can't see the toggle icon in this story

Reply:
@mohsenParvar I still can't see the icon here.

Reply:
it's on line 109 of story